shared/actions/login.js

```diff
@@ -600,6 +601,7 @@ function* _startLogin() {
 
 function _relogin({payload: {usernameOrEmail, passphrase}}: LoginGen.ReloginPayload) {
   return Saga.call(Saga.sequentially, [
+    Saga.put(LoginGen.createLoginError({error: ''})),
```

Comment:
so instead of explicitly doing this here you could add a new safeTakeEveryPure side effect like
```

const clearError = () => Saga.put(LoginGen.createLoginError({error: ''}))
...
 yield Saga.safeTakeLatest([LoginGen.startLogin, LoginGen.relogin], clearError)
```